packages/module/src/Hooks/useDataViewPagination.ts

```diff
@@ -0,0 +1,4 @@
+export interface DataViewPaginationProps {
+    page?: number;
```

Comment:
This should not be optional.

Reply:
@Hyperkid123 do you think we want to make perPage mandatory as well? I wonder if our implementation should contain a default value for that or if we want to let that up to the user of our component

Reply:
I don't think we will be able to "guess" defaults that would make sense.